tests/w3c_testsuite.rs

```diff
@@ -39,15 +41,15 @@ fn get_remote_file(url: &str) -> Result<String> {
     Ok(content)
 }
 
-fn parse_turtle(url: &str, data: &str) -> Result<Graph> {
-    GraphParser::from_format(GraphFormat::Turtle)
+fn parse_turtle(url: &str, data: &str) -> Result<Dataset> {
+    RdfParser::from_format(RdfFormat::Turtle)
         .with_base_iri(url)?
-        .read_triples(data.as_bytes())?
-        .collect::<std::result::Result<_, _>>()
+        .parse_read(data.as_bytes())
```

Comment:
To get back a graph:
```suggestion
        .parse_read(data.as_bytes())
        .map(|r| Ok(r?.into()))
```

Reply:
Hmm.. exactly like this it seems not work ...
What supports conversion into Graph, exactly?